### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by claucece**

### CODE REVIEW

File: chat/coyim/Makefile [hunk 1]
```diff
@@ -7,8 +7,8 @@ GITHUB_TAG=	v${PKGVERSION_NOREV}
 EXTRACT_USING=	bsdtar
 
 MAINTAINER=	fox@NetBSD.org
-HOMEPAGE=	https://github.com/coyim/coyim
-COMMENT=	Safe and secure chat client
+HOMEPAGE=	https://coy.im/
+COMMENT=	A safe and secure by default chat client
```
Comment (fraggerfox) [inline chat/coyim/Makefile]: Due to the rules about `COMMENT` in pkgsrc, a comment should not start with "A" / "An" or "The".

https://github.com/rillig/pkglint/blob/d5ba7e686c57cfa8d31b8e6d68804c480a32fc22/codewalk.md#line

**Review by fraggerfox — CHANGES_REQUESTED** [review]
If you can remove the "A" from the comment, it should comply with the pkgsrc `COMMENT` description guidelines. Then I can go ahead and merge these changes into the repository.

The rest looks good.

Thanks!!

File: chat/coyim/Makefile (hunk 1 above)
Comment (claucece) [reply chat/coyim/Makefile]: Changed ;)

Comment (fraggerfox) [reply chat/coyim/Makefile]: LGTM, pulling in the changes.

Thank you.

**Pull request merged**